rabbitmq-setup.sh

```diff
@@ -0,0 +1,17 @@
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list && sudo apt-get update && sudo apt-get install rabbitmq-server
+sudo apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev
+cd /tmp/
+git clone https://github.com/alanxz/rabbitmq-c
+mkdir rabbitmq-c/build && cd rabbitmq-c/build
+cmake ..
+cmake --build .
+sudo cp librabbitmq/*.a /usr/local/lib/
+sudo cp librabbitmq/*.so* /usr/local/lib/
+cd /tmp/
```

Comment:
Again, for readbility let's put a new line between `cd /tmp/` and the line above it.